README.md

```diff
@@ -570,18 +570,26 @@ type error interface {
 }
 ```
 
-A function that might return an error:
+Here's an example:
 ```go
-func doStuff() (int, error) {
+func sqrt(n float64) (sqrt_no float64, err error) {
```

Comment:
snake case variable name like `sqrt_no` is not encouraged in Go. With such a limited scope, I'd say `v` is clear enough.